templates/foundational_types.j2

```diff
@@ -0,0 +1,82 @@
+{% extends "base.j2" %}
+{% block title %}foundational types{% endblock %}
+
+{% block content %}
+
+{% filter convert_markdown %}
+Some of Lean's types are so primitive that they are available even in the `prelude`, and are not
+defined in any lean source files. This page provides some basic documentation on those types.
+
+
+For a more in depth explanation of Lean's type theory, refer to
+[TPiL](https://leanprover.github.io/theorem_proving_in_lean/dependent_type_theory.html).
```

Comment:
A pass at wordsmithery:

```suggestion
Some of Lean's types are not defined in any Lean source files (even the `prelude`) since they come from its foundational type theory. This page provides basic documentation for these types.

For a more in-depth explanation of Lean's type theory, refer to
[TPiL](https://leanprover.github.io/theorem_proving_in_lean/dependent_type_theory.html).
```